tests/unit/data/sboms/merged.bom.json

```diff
@@ -0,0 +1,278 @@
+{
+    "bomFormat": "CycloneDX",
+    "specVersion": "1.4",
+    "serialNumber": "urn:uuid:43840228-3fae-4d88-95aa-96512d2510dd",
+    "version": 1,
+    "metadata": {
+        "timestamp": "2023-05-03T18:19:41Z",
+        "tools": [
+            {
+                "vendor": "anchore",
+                "name": "syft",
+                "version": "0.47.0"
+            }
+        ],
+        "component": {
+            "bom-ref": "6b8edfe5f2756e0",
+            "type": "file",
+            "name": "/var/lib/containers/storage/vfs/dir/517aef0ffe20db360d19aa475dbbfbe03f452f53403881a31f9a475c83af788b"
+        }
+    },
```

Comment:
Observation: we prefer the Syft SBOM for everything except `components` (for those, we merge but prefer cachi2)

The merging is meant to be a temporary solution, right? Otherwise there's some more to think about IMO, like mismatched specVersion (note that 1.5 is already released)

Reply:
Yeah, I think adding Cachi2 components to Syft's SBOM makes more sense, since Syft is the default one now (that exists in all pipelines regardless of them being hermetic).

Syft is still using spec 1.4 even in the lastest release, though  =/

Reply:
Using 1.4 is fine, the problem would be if we bump cachi2's specVersion, use some new features but keep 1.4 in the merged SBOM

Reply:
Taking the higher of the two specVersions would probably be fine, but again, I don't really need it in a temporary solution

Reply:
Hopefully we will have a better solution for multiple SBOMs by then.